src/RoleSelect.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Vyuldashev\NovaPermission;
+
+use Illuminate\Support\Collection;
+use Laravel\Nova\Fields\Select;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Spatie\Permission\PermissionRegistrar;
+use Spatie\Permission\Traits\HasPermissions;
+
+class RoleSelect extends Select
```

Comment:
Please, add docs to readme of when/why and how to use it

Reply:
Done! :)